tests/output/macros.rs

```diff
@@ -364,3 +367,60 @@ async fn test_ocptv_log_fatal_in_step() -> Result<()> {
     })
     .await
 }
+
+#[tokio::test]
+async fn test_ocptv_diagnosis_pass_in_step() -> Result<()> {
+    let expected = json!({
+        "testStepArtifact": {
+            "diagnosis": {
+                    "verdict": "verdict",
+                    "type": "PASS",
+                }
+        },
+        "sequenceNumber": 3
```

Comment:
looks like all of these are missing timestamps and the tests are incorrectly passing; i'll fix this later, because it's a remnant of assert_json_include (or you can if you want in this PR)